pypixplore/skeleton.py

```diff
@@ -44,7 +44,7 @@ def parse_args(args):
       :obj:`argparse.Namespace`: command line parameters namespace
     """
     parser = argparse.ArgumentParser(
-        description="Explore Python Package Index")
+        description="Copyright (C) 2017 Flavio C. Coelho\nThis program comes with ABSOLUTELY NO WARRANTY;\nThis is free software, and you are welcome to redistribute it under certain conditions;\nExplore Python Package Index")
```

Comment:
This part "under certain conditions" is not correct, we are not showing the conditions for our users.